Cargo.toml

```diff
@@ -1,22 +1,22 @@
 [package]
 name = "phoenix-core"
-version = "0.10.0"
+version = "0.11.0-rc.0"
 authors = ["zer0 <matteo@dusk.network>", "Victor Lopez <victor@dusk.network"]
 edition = "2018"
 
 [dependencies]
-rand_core = "0.5"
+rand_core = "0.6"
 dusk-bytes = "0.1"
-dusk-bls12_381 = {version = "0.6", default-features = false}
-dusk-jubjub = {version = "0.8", default-features = false}
-dusk-poseidon = {version = "0.20", default-features = false}
-dusk-pki = {version = "0.6", default-features = false}
-canonical = {version = "0.5", optional = true}
-canonical_derive = {version = "0.5", optional = true}
+dusk-bls12_381 = {version="0.8", default-features=false}
+dusk-jubjub = {version="0.10", default-features=false}
+dusk-poseidon = {version="0.21.0-rc", default-features=false}
+dusk-pki = {version="0.7.0-rc", default-features=false}
+canonical = {version="0.6", optional=true}
+canonical_derive = {version="0.6", optional=true}
 
 [dev-dependencies]
 assert_matches = "1.3"
-rand = "0.7"
+rand = "0.8"
```

Comment:
We shouldn't have `rand` in dev-dependecies since `rand-core` should suffice. All the core libraries like bls and jubjub now are using `rand-core` and `OsRng` so we should be good to go.
If there is some other libraries used as deps here that still requires `rand` we should probably update them first and `phoenix-core` later.

Reply:
Fixed